src/components/Projects/index.js

```diff
@@ -27,6 +28,11 @@ import Chip from '@material-ui/core/Chip';
         slug
         bookmarked
         gitBranch
+        features(showDeployed: true, params: { limit: 25 }) {
```

Comment:
ShowDeployed should be false since I assume available features means new. Also, we may want to call it "new" instead of available since that's what the categorization already.